src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const RENT_PER_DAY = 40;
+
+  let discount = 0;
+
+  if (days >= SHORT_TERM) {
+    discount = SHORT_TERM_DISCOUNT;
+  }
+
+  if (days >= LONG_TERM) {
+    discount = LONG_TERM_DISCOUNT;
+  }
```

Comment:
Consider simplifying the conditionals by using `if` with `return`. As per the checklist, it's recommended to use `if` with `return` over `if else` to simplify the logic. Here, you can return the total amount as soon as the appropriate discount has been determined, without the need for an `else` or additional `if` statements.